crates/cxx-qt-build/src/lib.rs

```diff
@@ -476,6 +476,7 @@ impl CxxQtBuilder {
         if let Some(value) = value {
             println!("cargo:rustc-cfg={key}=\"{value}\"");
         } else {
+            println!("cargo:rustc-check-cfg=cfg({key})");
```

Comment:
I don't think this is quite the right spot?
This will only be emitted when the config variable is actually defined.
But we need to emit check-cfg commands for every variable we **may** define.

This is quite annoying to implement though, so I'm not sure where the best place to put it would be...

Reply:
With your current implementation I don't think we need this here at all, right?
You're already declaring all possible versions of qt_version_at_least and _major.

Reply:
Oops that was an old version, that line should be removed

Reply:
Fixed :-)